src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -260,4 +258,11 @@
       </cd-info-card>
     </div>
   </cd-info-group>
+
+  <ng-template #logsLink>
+    <ng-container *ngIf="permissions.log.read">
+      <p class="logs-link"
+         i18n>&rarr; See <a routerLink="/logs">Logs</a> for more details.</p>
```

Comment:
Not very familiar with Angular tricks, but isn't there any way that components provide anchors/links to them (to avoid hardcoded URNs)? I know that this has also been extensively done in the nav-bar component too. Just out of curiosity from SW design PoV.

Reply:
AFAIK there is no such feature.